README.md

```diff
@@ -34,6 +34,9 @@ Check out [the docs](http://hexdocs.pm/ex_machina/ExMachina.html) for more detai
 Define factories:
 
 ```elixir
+# You could put this in `test/support/factory.ex` if you use it for tests only,
```

Comment:
Tried a scripted file as well, didn't seem to work. I don't think you can use a module defined in a script from another script – I think the stuff you use elsewhere needs to be compiled, and scripts are just things that can use compiled things but can't be used themselves.

Experiments seem to bear that out but I could be wrong.

I force pushed an update, suggesting adding the `test/support` dir to the path if it's not there.